<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Sequencing | Microbiology</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="This section focuses on sequencing, specifically 16S rRNA gene sequencing, as a powerful method for bacterial identification. It covers the theory, interpretation, and application of this widely...">
<meta name="generator" content="bookdown 0.42 with bs4_book()">
<meta property="og:title" content="Sequencing | Microbiology">
<meta property="og:type" content="book">
<meta property="og:description" content="This section focuses on sequencing, specifically 16S rRNA gene sequencing, as a powerful method for bacterial identification. It covers the theory, interpretation, and application of this widely...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Sequencing | Microbiology">
<meta name="twitter:description" content="This section focuses on sequencing, specifically 16S rRNA gene sequencing, as a powerful method for bacterial identification. It covers the theory, interpretation, and application of this widely...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.8.0/transition.js"></script><script src="libs/bs3compat-0.8.0/tabs.js"></script><script src="libs/bs3compat-0.8.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Microbiology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Preanalytic</li>
<li><a class="" href="collection-transport.html">Collection &amp; Transport</a></li>
<li><a class="" href="patient-id-labeling.html">Patient ID &amp; Labeling</a></li>
<li><a class="" href="specimen-collection.html">Specimen Collection</a></li>
<li><a class="" href="sterile-technique.html">Sterile Technique</a></li>
<li><a class="" href="blood-culture.html">Blood Culture</a></li>
<li><a class="" href="sites-with-flora.html">Sites with Flora</a></li>
<li><a class="" href="transport-systems.html">Transport Systems</a></li>
<li><a class="" href="transport-devices-1.html">Transport Devices</a></li>
<li><a class="" href="atmosphere-1.html">Atmosphere</a></li>
<li><a class="" href="time-temperature.html">Time &amp; Temperature</a></li>
<li><a class="" href="processing.html">Processing</a></li>
<li><a class="" href="prioritization-rejection.html">Prioritization &amp; Rejection</a></li>
<li><a class="" href="biosafety-cabinet-ppe.html">Biosafety Cabinet &amp; PPE</a></li>
<li><a class="" href="preparation-methods.html">Preparation Methods</a></li>
<li><a class="" href="concentration-1.html">Concentration</a></li>
<li><a class="" href="digestion-decontamination.html">Digestion &amp; Decontamination</a></li>
<li><a class="" href="cross-contamination.html">Cross-Contamination</a></li>
<li><a class="" href="sterile-technique-2.html">Sterile Technique</a></li>
<li><a class="" href="tissue-homogenization-1.html">Tissue Homogenization</a></li>
<li><a class="" href="dnarna-extraction-1.html">DNA/RNA extraction</a></li>
<li><a class="" href="media-1.html">Media</a></li>
<li><a class="" href="types.html">Types</a></li>
<li><a class="" href="components.html">Components</a></li>
<li><a class="" href="application-use.html">Application &amp; Use</a></li>
<li><a class="" href="selection.html">Selection</a></li>
<li><a class="" href="fastidious-species.html">Fastidious Species</a></li>
<li><a class="" href="mycobacteria-fungi.html">Mycobacteria &amp; Fungi</a></li>
<li><a class="" href="inoculation.html">Inoculation</a></li>
<li><a class="" href="quantitative.html">Quantitative</a></li>
<li><a class="" href="semiquantitative.html">Semiquantitative</a></li>
<li><a class="" href="automated-plating.html">Automated Plating</a></li>
<li><a class="" href="incubation-conditions-1.html">Incubation Conditions</a></li>
<li><a class="" href="slide-preparation.html">Slide Preparation</a></li>
<li><a class="" href="stains.html">Stains</a></li>
<li><a class="" href="gram.html">Gram</a></li>
<li><a class="" href="acid-fast.html">Acid-fast</a></li>
<li><a class="" href="modified-acid-fast.html">Modified Acid-fast</a></li>
<li><a class="" href="koh-calcofluor-white.html">KOH &amp; Calcofluor-White</a></li>
<li><a class="" href="trichrome.html">Trichrome</a></li>
<li><a class="" href="giemsa.html">Giemsa</a></li>
<li><a class="" href="acridine-orange-1.html">Acridine Orange</a></li>
<li><a class="" href="modified-trichrome-1.html">Modified Trichrome</a></li>
<li class="book-part">Bacteriology</li>
<li><a class="" href="blood-bone-marrow.html">Blood &amp; Bone Marrow</a></li>
<li><a class="" href="sample-sources.html">Sample Sources</a></li>
<li><a class="" href="continuous-monitoring.html">Continuous Monitoring</a></li>
<li><a class="" href="rapid-id-methods.html">Rapid ID Methods</a></li>
<li><a class="" href="skin-flora.html">Skin Flora</a></li>
<li><a class="" href="colony-morphology-id.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="agents-of-endocarditis.html">Agents of Endocarditis</a></li>
<li><a class="" href="agents-of-bone-marrow.html">Agents of Bone Marrow</a></li>
<li><a class="" href="organism-pathogenicity-1.html">Organism Pathogenicity</a></li>
<li><a class="" href="cerebrospinal-fluid.html">Cerebrospinal Fluid</a></li>
<li><a class="" href="sample-sources-1.html">Sample Sources</a></li>
<li><a class="" href="colony-morphology-id-1.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="agents-of-shunt.html">Agents of Shunt</a></li>
<li><a class="" href="laboratory-results.html">Laboratory Results</a></li>
<li><a class="" href="antigens-molecular.html">Antigens &amp; Molecular</a></li>
<li><a class="" href="organism-pathogenicity-3.html">Organism Pathogenicity</a></li>
<li><a class="" href="sterile-fluids.html">Sterile Fluids</a></li>
<li><a class="" href="sample-sources-2.html">Sample Sources</a></li>
<li><a class="" href="indeginous-surface-organisms.html">Indeginous Surface Organisms</a></li>
<li><a class="" href="colony-morphology-id-2.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="molecular-methods-2.html">Molecular Methods</a></li>
<li><a class="" href="organism-pathogenicity-5.html">Organism Pathogenicity</a></li>
<li><a class="" href="lower-respiratory.html">Lower Respiratory</a></li>
<li><a class="" href="sample-sources-3.html">Sample Sources</a></li>
<li><a class="" href="quantitative-reporting.html">Quantitative Reporting</a></li>
<li><a class="" href="oral-flora.html">Oral Flora</a></li>
<li><a class="" href="colony-morphology-id-3.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="community-associated.html">Community-Associated</a></li>
<li><a class="" href="healthcare-associated.html">Healthcare-Associated</a></li>
<li><a class="" href="cystic-fibrosis.html">Cystic fibrosis</a></li>
<li><a class="" href="molecular-methods-4.html">Molecular Methods</a></li>
<li><a class="" href="organism-pathogenicity-7.html">Organism Pathogenicity</a></li>
<li><a class="" href="upper-respiratory.html">Upper Respiratory</a></li>
<li><a class="" href="sample-sources-4.html">Sample Sources</a></li>
<li><a class="" href="indeginous-flora.html">Indeginous Flora</a></li>
<li><a class="" href="colony-morphology-id-4.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="pharyngitis-1.html">Pharyngitis</a></li>
<li><a class="" href="otitis-media-sinusitis.html">Otitis Media &amp; Sinusitis</a></li>
<li><a class="" href="molecular-methods-6.html">Molecular Methods</a></li>
<li><a class="" href="organism-pathogenicity-9.html">Organism Pathogenicity</a></li>
<li><a class="" href="gastrointestinal.html">Gastrointestinal</a></li>
<li><a class="" href="colony-morphology-id-5.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="molecular-methods-8.html">Molecular Methods</a></li>
<li><a class="" href="serotyping.html">Serotyping</a></li>
<li><a class="" href="organism-pathogenicity-11.html">Organism Pathogenicity</a></li>
<li><a class="" href="h.-pylori-detection.html">H. pylori Detection</a></li>
<li><a class="" href="skin-soft-tissue-bone.html">Skin, Soft Tissue, &amp; Bone</a></li>
<li><a class="" href="specimen-sources-3.html">Specimen Sources</a></li>
<li><a class="" href="indigenous-flora.html">Indigenous Flora</a></li>
<li><a class="" href="colony-morphology-id-6.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="skin-1.html">Skin</a></li>
<li><a class="" href="soft-tissue.html">Soft Tissue</a></li>
<li><a class="" href="bone.html">Bone</a></li>
<li><a class="" href="bite-wound.html">Bite Wound</a></li>
<li><a class="" href="zoonotic.html">Zoonotic</a></li>
<li><a class="" href="organism-pathogenicity-13.html">Organism Pathogenicity</a></li>
<li><a class="" href="genital-tract.html">Genital Tract</a></li>
<li><a class="" href="specimen-sources-5.html">Specimen Sources</a></li>
<li><a class="" href="indigenous-organisms.html">Indigenous Organisms</a></li>
<li><a class="" href="vaginitis.html">Vaginitis</a></li>
<li><a class="" href="culture-molecular.html">Culture &amp; Molecular</a></li>
<li><a class="" href="organism-pathogenicity-15.html">Organism Pathogenicity</a></li>
<li><a class="" href="urine.html">Urine</a></li>
<li><a class="" href="specimen-sources-6.html">Specimen Sources</a></li>
<li><a class="" href="colony-morphology-id-7.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="correlation-of-colony-counts.html">Correlation of Colony Counts</a></li>
<li><a class="" href="correlation-of-cultures.html">Correlation of Cultures</a></li>
<li><a class="" href="identification-methods.html">Identification Methods</a></li>
<li><a class="" href="colony-morphology-1.html">Colony Morphology</a></li>
<li><a class="" href="rapid-presumptive-id.html">Rapid Presumptive ID</a></li>
<li><a class="" href="conventional-biochemical-id.html">Conventional Biochemical ID</a></li>
<li><a class="" href="commercial-kits.html">Commercial Kits</a></li>
<li><a class="" href="automated-methods.html">Automated Methods</a></li>
<li><a class="" href="maldi-tof-ms.html">MALDI-TOF MS</a></li>
<li><a class="" href="molecular-methods-10.html">Molecular Methods</a></li>
<li><a class="active" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="susceptibility-testing.html">Susceptibility Testing</a></li>
<li><a class="" href="susceptibility-testing-1.html">Susceptibility Testing</a></li>
<li><a class="" href="microbroth-dilution.html">Microbroth Dilution</a></li>
<li><a class="" href="disk-diffusion.html">Disk Diffusion</a></li>
<li><a class="" href="gradient-diffusion.html">Gradient Diffusion</a></li>
<li><a class="" href="phenotypic-resistance.html">Phenotypic Resistance</a></li>
<li><a class="" href="mechanisms-of-action.html">Mechanisms of Action</a></li>
<li><a class="" href="genetic-determinants.html">Genetic Determinants</a></li>
<li><a class="" href="intrinsic-resistance-patterns.html">Intrinsic Resistance Patterns</a></li>
<li><a class="" href="antimicrobials-reporting.html">Antimicrobials Reporting</a></li>
<li><a class="" href="mrsamssa-vre-esblcre.html">MRSA/MSSA, VRE, ESBL/CRE</a></li>
<li><a class="" href="specimen-sources-8.html">Specimen Sources</a></li>
<li><a class="" href="culture-methods-1.html">Culture Methods</a></li>
<li><a class="" href="molecular-methods-13.html">Molecular Methods</a></li>
<li><a class="" href="bsl-3-pathogens.html">BSL-3 Pathogens</a></li>
<li><a class="" href="specimen-sources-10.html">Specimen sources</a></li>
<li><a class="" href="colony-morphology-id-8.html">Colony Morphology &amp; ID</a></li>
<li><a class="" href="laboratory-response-network.html">Laboratory Response Network</a></li>
<li><a class="" href="organism-pathogenicity-17.html">Organism Pathogenicity</a></li>
<li class="book-part">Mycobacteriology</li>
<li><a class="" href="procedures.html">Procedures</a></li>
<li><a class="" href="specimen-sources-11.html">Specimen Sources</a></li>
<li><a class="" href="pathogens-diseases.html">Pathogens &amp; Diseases</a></li>
<li><a class="" href="stain-morph-growth.html">Stain, Morph, Growth</a></li>
<li><a class="" href="major-pathogens.html">Major Pathogens</a></li>
<li><a class="" href="common-contaminant.html">Common Contaminant</a></li>
<li><a class="" href="less-common-pathogens.html">Less Common Pathogens</a></li>
<li><a class="" href="identification-methods-1.html">Identification Methods</a></li>
<li><a class="" href="molecular-methods-14.html">Molecular Methods</a></li>
<li><a class="" href="antimicrobial-therapy.html">Antimicrobial Therapy</a></li>
<li><a class="" href="m.-tuberculosis.html">M. tuberculosis</a></li>
<li><a class="" href="mai-m.-kansasii-m.-marinum.html">MAI, M. kansasii, M. marinum</a></li>
<li><a class="" href="rapid-growers.html">Rapid Growers</a></li>
<li><a class="" href="susceptibility-testing-2.html">Susceptibility testing</a></li>
<li><a class="" href="broth-microdilution-1.html">Broth Microdilution</a></li>
<li><a class="" href="critical-concentration-1.html">Critical Concentration</a></li>
<li><a class="" href="resistance-markers.html">Resistance Markers</a></li>
<li><a class="" href="organism-pathogenicity-18.html">Organism pathogenicity</a></li>
<li class="book-part">Virology</li>
<li><a class="" href="procedures-1.html">Procedures</a></li>
<li><a class="" href="specimen-sources-13.html">Specimen Sources</a></li>
<li><a class="" href="major-pathogens-1.html">Major pathogens</a></li>
<li><a class="" href="respiratory.html">Respiratory</a></li>
<li><a class="" href="vesicles-lesions.html">Vesicles &amp; Lesions</a></li>
<li><a class="" href="cervical-cancer.html">Cervical Cancer</a></li>
<li><a class="" href="meningitis-1.html">Meningitis</a></li>
<li><a class="" href="gastroenteritis-1.html">Gastroenteritis</a></li>
<li><a class="" href="immunocompromised.html">Immunocompromised</a></li>
<li><a class="" href="hepatitis-1.html">Hepatitis</a></li>
<li><a class="" href="hiv-1.html">HIV</a></li>
<li><a class="" href="other-viruses-1.html">Other Viruses</a></li>
<li><a class="" href="direct-detection.html">Direct Detection</a></li>
<li><a class="" href="antiviral-agents.html">Antiviral Agents</a></li>
<li class="book-part">Parasitology</li>
<li><a class="" href="procedures-2.html">Procedures</a></li>
<li><a class="" href="specimen-sources-14.html">Specimen Sources</a></li>
<li><a class="" href="major-pathogens-2.html">Major Pathogens</a></li>
<li><a class="" href="blood-tissue-protozoa.html">Blood &amp; Tissue Protozoa</a></li>
<li><a class="" href="intestinal-urogenital-protozoa.html">Intestinal &amp; Urogenital Protozoa</a></li>
<li><a class="" href="intestinal-tissue-helminths.html">Intestinal &amp; Tissue Helminths</a></li>
<li><a class="" href="brain.html">Brain</a></li>
<li><a class="" href="insects-arthropods.html">Insects &amp; Arthropods</a></li>
<li><a class="" href="additional-parasites-1.html">Additional Parasites</a></li>
<li><a class="" href="micro-macro-id.html">Micro &amp; Macro ID</a></li>
<li><a class="" href="direct-detection-1.html">Direct Detection</a></li>
<li><a class="" href="culture-7.html">Culture</a></li>
<li class="book-part">Mycology</li>
<li><a class="" href="procedures-3.html">Procedures</a></li>
<li><a class="" href="specimen-sources-16.html">Specimen Sources</a></li>
<li><a class="" href="superficial.html">Superficial</a></li>
<li><a class="" href="deep-systemic.html">Deep &amp; Systemic</a></li>
<li><a class="" href="systemic.html">Systemic</a></li>
<li><a class="" href="major-pathogens-3.html">Major Pathogens</a></li>
<li><a class="" href="yeast.html">Yeast</a></li>
<li><a class="" href="dimorphic-fungi-1.html">Dimorphic Fungi</a></li>
<li><a class="" href="dermatophytes-1.html">Dermatophytes</a></li>
<li><a class="" href="mucorales.html">Mucorales</a></li>
<li><a class="" href="hyaline-molds-1.html">Hyaline Molds</a></li>
<li><a class="" href="dematiaceous-molds-1.html">Dematiaceous Molds</a></li>
<li><a class="" href="pneumocystis-jirovecii-1.html">Pneumocystis Jirovecii</a></li>
<li><a class="" href="microsporidium.html">Microsporidium</a></li>
<li><a class="" href="morphology-growth.html">Morphology &amp; Growth</a></li>
<li><a class="" href="microscopic-id.html">Microscopic ID</a></li>
<li><a class="" href="direct-detection-2.html">Direct Detection</a></li>
<li><a class="" href="other-id-methods.html">Other ID Methods</a></li>
<li><a class="" href="antifungal-susceptibility.html">Antifungal Susceptibility</a></li>
<li><a class="" href="manual-automated.html">Manual &amp; Automated</a></li>
<li><a class="" href="antifungal-classes.html">Antifungal Classes</a></li>
<li><a class="" href="intrinsic-resistance-1.html">Intrinsic Resistance</a></li>
<li class="book-part">Serology</li>
<li><a class="" href="procedures-4.html">Procedures</a></li>
<li><a class="" href="immunoglobulin-response.html">Immunoglobulin Response</a></li>
<li><a class="" href="ag-ab-interactions.html">Ag-Ab Interactions</a></li>
<li><a class="" href="principles.html">Principles</a></li>
<li><a class="" href="testing-methods.html">Testing Methods</a></li>
<li><a class="" href="viral-pathogens.html">Viral Pathogens</a></li>
<li><a class="" href="agents-of-encephalitis.html">Agents of Encephalitis</a></li>
<li><a class="" href="t.-pallidum-b.-burgdorferi.html">T. pallidum &amp; B. burgdorferi</a></li>
<li><a class="" href="tuberculosis.html">Tuberculosis</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Microbiology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="sequencing" class="section level1 unnumbered">
<h1>Sequencing<a class="anchor" aria-label="anchor" href="#sequencing"><i class="fas fa-link"></i></a>
</h1>
<p>This section focuses on sequencing, specifically 16S rRNA gene sequencing, as a powerful method for bacterial identification. It covers the theory, interpretation, and application of this widely used molecular technique</p>
<div id="theory-unlocking-bacterial-identity-through-the-ribosomes-code" class="section level2 unnumbered">
<h2>
<strong>Theory: Unlocking Bacterial Identity Through the Ribosome’s Code</strong><a class="anchor" aria-label="anchor" href="#theory-unlocking-bacterial-identity-through-the-ribosomes-code"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>What is 16S rRNA Gene Sequencing?</strong>
<ul>
<li>16S rRNA gene sequencing is a <em>molecular technique</em> used for bacterial identification and phylogenetic analysis</li>
<li>It involves sequencing a highly <em>conserved</em> (present in all bacteria) and <em>highly variable</em> (different between species) region of the 16S ribosomal RNA (rRNA) gene</li>
<li>The <em>unique sequence</em> of the 16S rRNA gene is used to identify the bacterium to the <em>species level</em>
</li>
</ul>
</li>
<li>
<strong>Why Use 16S rRNA Gene Sequencing?</strong>
<ul>
<li>
<strong>Universal:</strong> Present in <em>all bacteria</em>, allowing for identification of a broad range of organisms</li>
<li>
<strong>Highly Informative:</strong> The 16S rRNA gene contains <em>conserved regions</em> for primer design and <em>variable regions</em> for species-level differentiation</li>
<li>
<strong>Culture-Independent:</strong> Can identify bacteria <em>without the need for prior culture</em> (e.g., in complex environments)</li>
<li>
<strong>High Accuracy:</strong> Provides <em>accurate and reliable</em> species identification</li>
<li>
<strong>Phylogenetic Analysis:</strong> Allows for the <em>determination of evolutionary relationships</em> between bacteria</li>
<li>
<strong>Identification of Non-Culturable Bacteria:</strong> Useful for identifying bacteria that are difficult or impossible to culture</li>
</ul>
</li>
<li>
<strong>The 16S rRNA Gene</strong>
<ul>
<li>The 16S rRNA gene is a <em>highly conserved gene</em> found in all bacteria</li>
<li>It encodes for the <em>16S ribosomal RNA</em>, a structural component of the bacterial ribosome</li>
<li>The gene contains <em>conserved regions</em> (highly similar across bacterial species) and <em>variable regions</em> (different between species)</li>
<li>The <em>conserved regions</em> are used for designing <em>universal primers</em> for PCR amplification</li>
<li>The <em>variable regions</em> are used for <em>species-level identification</em>
</li>
</ul>
</li>
<li>
<strong>General Principle of 16S rRNA Gene Sequencing</strong>
<ol style="list-style-type: decimal">
<li>
<strong>DNA Extraction:</strong> Extract DNA from the bacterial sample</li>
<li>
<strong>PCR Amplification:</strong> Amplify the 16S rRNA gene using universal primers that bind to the conserved regions</li>
<li>
<strong>Sequencing:</strong> Determine the nucleotide sequence of the amplified 16S rRNA gene</li>
<li>
<strong>Sequence Analysis:</strong> Compare the sequence to a database of known 16S rRNA gene sequences</li>
<li>
<strong>Identification:</strong> Identify the bacterium based on the sequence match</li>
</ol>
</li>
</ul>
</div>
<div id="interpretation-decoding-the-genetic-fingerprint" class="section level2 unnumbered">
<h2>
<strong>Interpretation: Decoding the Genetic Fingerprint</strong><a class="anchor" aria-label="anchor" href="#interpretation-decoding-the-genetic-fingerprint"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Sequencing Results</strong>
<ul>
<li>
<strong>Sequence Data:</strong> The result of the sequencing reaction is a <em>DNA sequence</em> representing the 16S rRNA gene</li>
<li>
<strong>Sequence Alignment:</strong> The sequence is <em>aligned</em> with a database of known 16S rRNA gene sequences</li>
<li>
<strong>Percentage Identity:</strong> The <em>percentage identity</em> between the unknown sequence and the closest match in the database is calculated. This is the most important result</li>
<li>
<strong>Closest Match:</strong> The database provides the <em>closest matching species</em> based on the sequence alignment</li>
<li>
<strong>E-value:</strong> The <em>E-value</em> is a statistical value that estimates the number of hits one can expect to see by chance when searching a database of a particular size. A lower E-value indicates a more significant match</li>
<li>
<strong>Confidence Level:</strong> The identification is often reported with a <em>confidence level</em> (e.g., high, medium, low)</li>
<li>
<strong>Phylogenetic Tree:</strong> The results can be used to construct a <em>phylogenetic tree</em> showing the evolutionary relationships between the unknown bacterium and other bacteria</li>
</ul>
</li>
<li>
<strong>Interpreting the Percentage Identity</strong>
<ul>
<li>
<strong>Species-Level Identification:</strong> Generally, a <em>percentage identity of &gt;98-99%</em> is considered sufficient for <em>species-level identification</em>
</li>
<li>
<strong>Genus-Level Identification:</strong> A <em>percentage identity of 95-98%</em> may indicate <em>genus-level identification</em>
</li>
<li>
<strong>Unidentified:</strong> If the percentage identity is <em>low</em> (e.g., &lt;95%), the bacterium may be a <em>novel species</em> or may not be present in the database</li>
</ul>
</li>
<li>
<strong>Factors Affecting Accuracy</strong>
<ul>
<li>
<strong>Database Quality:</strong> The <em>accuracy and completeness</em> of the database are critical</li>
<li>
<strong>Sequence Quality:</strong> The <em>quality of the sequencing data</em> affects the accuracy of the identification</li>
<li>
<strong>Primer Specificity:</strong> The <em>primers used for PCR amplification</em> must be specific to the 16S rRNA gene</li>
<li>
<strong>Multiple Organisms:</strong> If the sample contains <em>multiple organisms</em>, the results may be ambiguous</li>
<li>
<strong>Strain Variation:</strong> <em>Strain variation</em> can affect the sequence, but the method is usually robust enough to identify down to the species level</li>
</ul>
</li>
</ul>
</div>
<div id="application-putting-knowledge-into-practice-7" class="section level2 unnumbered">
<h2>
<strong>Application: Putting Knowledge into Practice</strong><a class="anchor" aria-label="anchor" href="#application-putting-knowledge-into-practice-7"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Quality Control (QC)</strong>
<ul>
<li>
<strong>Control Strains:</strong> Use known positive and negative control organisms for each run</li>
<li>
<strong>Sequencing of the Control Strains:</strong> Sequence known species to check if the sequencing process and database are working correctly</li>
<li>
<strong>Sequence Quality Checks:</strong> Assess the <em>quality of the sequencing data</em> (e.g., read length, Phred scores)</li>
<li>
<strong>Documentation:</strong> Record QC results in a logbook or LIS</li>
<li>
<strong>Repeat if Necessary:</strong> If QC fails, repeat the sequencing run</li>
</ul>
</li>
<li>
<strong>Procedure</strong>
<ol style="list-style-type: decimal">
<li>
<strong>Sample Preparation:</strong> Prepare the bacterial sample (e.g., from a pure culture, from a clinical sample)</li>
<li>
<strong>DNA Extraction:</strong> Extract DNA from the sample using a suitable extraction method</li>
<li>
<strong>PCR Amplification:</strong> Amplify the 16S rRNA gene using universal primers</li>
<li>
<strong>PCR Product Purification:</strong> Purify the PCR product to remove excess primers and other contaminants</li>
<li>
<strong>Sequencing</strong>
<ul>
<li>Prepare the sequencing reaction</li>
<li>Load the sample into the sequencing instrument</li>
<li>Run the sequencing reaction</li>
</ul>
</li>
<li>
<strong>Sequence Analysis</strong>
<ul>
<li>Obtain the sequence data</li>
<li>Trim the sequence to remove low-quality regions</li>
<li>Align the sequence with a database</li>
<li>Determine the percentage identity and closest match</li>
<li>Generate a phylogenetic tree (optional)</li>
</ul>
</li>
<li>
<strong>Result Interpretation:</strong> Interpret the results based on the percentage identity and the closest match</li>
<li>
<strong>Documentation:</strong> Record the results in the LIS</li>
<li>
<strong>Correlation:</strong> Correlate the results with Gram stain, colony morphology, and other clinical information</li>
<li>
<strong>Reporting:</strong> Report the identification to the clinician</li>
</ol>
</li>
<li>
<strong>Examples of Applications</strong>
<ul>
<li><strong>Identification of Bacteria from Culture</strong></li>
<li><strong>Identification of Bacteria from Clinical Samples</strong></li>
<li><strong>Identification of Bacteria that are difficult to culture</strong></li>
<li><strong>Identification of Bacteria from environmental samples (e.g., water, soil)</strong></li>
<li><strong>Outbreak Investigations</strong></li>
<li><strong>Phylogenetic Studies</strong></li>
</ul>
</li>
<li>
<strong>Troubleshooting</strong>
<ul>
<li>
<strong>Poor Sequencing Results</strong>
<ul>
<li>
<strong>Low DNA Concentration:</strong> Ensure sufficient DNA for sequencing</li>
<li>
<strong>Poor DNA Quality:</strong> Ensure good DNA quality (e.g., no degradation)</li>
<li>
<strong>Contamination:</strong> Use clean reagents and maintain aseptic technique</li>
<li>
<strong>Primer Issues:</strong> Optimize primer design and reaction conditions</li>
</ul>
</li>
<li>
<strong>Ambiguous Identification</strong>
<ul>
<li>
<strong>Low Percentage Identity:</strong> The bacterium may be a novel species</li>
<li>
<strong>Multiple Closest Matches:</strong> The sample may contain a mixed population of bacteria</li>
<li>
<strong>Database Limitations:</strong> The bacterium may not be present in the database</li>
</ul>
</li>
<li>
<strong>Technical Issues</strong>
<ul>
<li>
<strong>Instrument Malfunction:</strong> Contact the sequencing instrument manufacturer for technical support</li>
<li>
<strong>Reagent Problems:</strong> Use fresh reagents and follow the manufacturer’s instructions</li>
<li>
<strong>Poor Sample Preparation:</strong> Ensure proper sample preparation techniques are followed</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div id="key-terms-96" class="section level2 unnumbered">
<h2>
<strong>Key Terms</strong><a class="anchor" aria-label="anchor" href="#key-terms-96"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>16S rRNA Gene Sequencing:</strong> A molecular technique for bacterial identification based on sequencing the 16S rRNA gene</li>
<li>
<strong>16S rRNA Gene:</strong> A gene that encodes for the 16S ribosomal RNA</li>
<li>
<strong>Ribosome:</strong> A cellular structure responsible for protein synthesis</li>
<li>
<strong>Conserved Region:</strong> A region of a gene or protein that is highly similar across different species</li>
<li>
<strong>Variable Region:</strong> A region of a gene or protein that is different between different species</li>
<li>
<strong>DNA Extraction:</strong> The process of isolating DNA from a sample</li>
<li>
<strong>PCR (Polymerase Chain Reaction):</strong> A technique for amplifying specific DNA sequences</li>
<li>
<strong>Primer:</strong> A short DNA sequence that initiates DNA synthesis</li>
<li>
<strong>Sequencing:</strong> Determining the order of nucleotides in a DNA sequence</li>
<li>
<strong>Nucleotide:</strong> The basic building block of DNA and RNA</li>
<li>
<strong>Sequence Alignment:</strong> Comparing two or more DNA sequences to identify similarities and differences</li>
<li>
<strong>Percentage Identity:</strong> The percentage of identical nucleotides between two DNA sequences</li>
<li>
<strong>Closest Match:</strong> The bacterial species that has the highest percentage identity with the unknown sequence</li>
<li>
<strong>E-value:</strong> A statistical value that estimates the number of hits one can expect to see by chance</li>
<li>
<strong>Phylogenetic Tree:</strong> A diagram that shows the evolutionary relationships between organisms</li>
<li>
<strong>Quality Control (QC):</strong> Procedures used to monitor and ensure the reliability of laboratory testing</li>
<li>
<strong>Control Strains:</strong> Known organisms used as positive and negative controls</li>
<li>
<strong>Gram Stain:</strong> A differential staining technique used to classify bacteria</li>
<li>
<strong>Colony Morphology:</strong> The visual characteristics of bacterial colonies on solid media</li>
<li>
<strong>LIS (Laboratory Information System):</strong> A computer system used to manage laboratory data</li>
<li>
<strong>Aseptic Technique:</strong> Procedures used to prevent contamination</li>
<li>
<strong>Amplification:</strong> The process of making multiple copies of a DNA sequence</li>
<li>
<strong>Taxonomy:</strong> The science of classifying organisms</li>
<li>
<strong>Phylogeny:</strong> The evolutionary history of a species or group of species</li>
<li>
<strong>Annotation:</strong> The process of identifying and describing the features of a genome</li>
<li>
<strong>Strain:</strong> A genetic variant or subtype of a species</li>
<li>
<strong>Genome:</strong> The complete set of genetic material in an organism</li>
<li>
<strong>Read Length:</strong> The number of nucleotides in a DNA sequence read</li>
<li>
<strong>Phred Score:</strong> A measure of the quality of a nucleotide base call in a DNA sequence</li>
<li>
<strong>Trim:</strong> To remove low-quality regions from a DNA sequence</li>
<li>
<strong>Consensus Sequence:</strong> A single sequence that represents the most likely sequence for a given region of DNA</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();
  
  // hide those labes that take up multiple lines
  l = [];
  function hide_label() {
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase" 
          && 
          !elem.classList
          .contains('book-part') 
          ? l.push(elem) 
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    count = 0;
    prev_elem = '';
    l.forEach((elem) => {
      if(count === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_elem.classList.add('hide_label');
        } else {
          prev_elem.classList.remove('hide_label');
        }
      }
      prev_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="molecular-methods-10.html">Molecular Methods</a></div>
<div class="next"><a href="susceptibility-testing.html">Susceptibility Testing</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#sequencing">Sequencing</a></li>
<li><a class="nav-link" href="#theory-unlocking-bacterial-identity-through-the-ribosomes-code">Theory: Unlocking Bacterial Identity Through the Ribosome’s Code</a></li>
<li><a class="nav-link" href="#interpretation-decoding-the-genetic-fingerprint">Interpretation: Decoding the Genetic Fingerprint</a></li>
<li><a class="nav-link" href="#application-putting-knowledge-into-practice-7">Application: Putting Knowledge into Practice</a></li>
<li><a class="nav-link" href="#key-terms-96">Key Terms</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Microbiology/blob/blob/main/Rmd/02-J-08.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Microbiology/blob/edit/main/Rmd/02-J-08.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3" style="text-align: center; flex: 100%; max-width: 100%;">
    <p>Written by Brett M. Rice | Updated 2025-03-30 | Built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package</p>
  </div>

  <!--div class="col-12 col-md-6 mt-3">
    <p>Built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package</p>
  </div-->

</div></div>
</footer>
</body>
</html>
